src/dotnet/Chat.UI.Blazor/Components/ContactListNavbarWidget.razor

```diff
@@ -73,11 +73,15 @@
 
 <NavbarGroup Title="Contacts" Class="navbar-contacts" Id="contacts">
     <ChildContent>
-        <TabPanel BottomHill="true" ActiveTabChanged="OnActiveTabChanged" TabsClass="wide-left-panel-tabs">
-            @RenderTab(ContactGroup.All, "All")
-            @RenderTab(ContactGroup.Recent, "Recent")
-            @RenderTab(ContactGroup.Favorites, "Favorites")
-        </TabPanel>
+        <div class="-flex-y w-full h-full">
+            <ListeningChatsNavbarWidget/>
```

Comment:
It's bit a bit counterintuitive that reerse order of components is used (-flex-y and ListeningChatsNavbarWidget goes first). Why is done that way?

Reply:
With -flex-y it is easier and more convenient to press the resizable block to the bottom of the page